app/views/declaration/officeOfExit/office_of_exit_supplementary.scala.html

```diff
@@ -14,48 +14,73 @@
+)
 
 @(mode: Mode, form: Form[OfficeOfExitSupplementary])(implicit request: JourneyRequest[_], messages: Messages)
 
 @tariffBodyDesc = { @Html(messages("declaration.type.consignmentTariffText", components.details_content_link())) }
+@tariffBody = {
+@components.tariff_body_partial(
+    messages("supplementary.representative-details.help-item1"),
+    messages("supplementary.representative-details.help-item2")
+)(tariffBodyDesc)
+}
+@tariff = { @components.details(messages("site.details.summary_text_these")){ @tariffBody } }
+
+@autocomplete = {
+@components.fields.field_accessible_autocomplete(
+    field = form("officeId"),
+    label = messages("declaration.officeOfExit"),
+    labelClass = Some("form-label-bold"),
+    hintText = Some(messages("declaration.officeOfExit.hint")),
+    messages("declaration.officeOfExit.empty"),
+    AutoCompleteItem.fromOfficeOfExit(OfficeOfExits.all)
+)
+}
+
+@govukLayout(
+    title = Title("declaration.officeOfExit.title", "declaration.summary.locations.header"),
+    backButton = Some(BackButton(messages("site.back"), Navigator.backLink(OfficeOfExitSupplementary, mode)))){
+
+    @formHelper(action = controllers.declaration.routes.OfficeOfExitController.saveOffice(mode), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
+
+        @sectionHeader(messages("declaration.summary.locations.header"))
 
-@tariffBody = { @components.tariff_body_partial(messages("supplementary.representative-details.help-item1"), messages("supplementary.representative-details.help-item2"))(tariffBodyDesc)}
-
-@main_template(
-    title = Title("declaration.officeOfExit.title", "declaration.summary.locations.header")
-) {
-    @helper.form(routes.OfficeOfExitController.saveOffice(mode), 'autoComplete -> "off") {
-        @helper.CSRF.formField
-
-        @components.back_link(Navigator.backLink(OfficeOfExitSupplementary, mode), mode)
-
-        @components.error_summary(form.errors)
-
-        @components.heading(
-            sectionHeader = messages("declaration.summary.locations.header"),
-            title = messages("declaration.officeOfExit.title"))
-
-        @components.fields.field_autocomplete(
-                  field = form("officeId"),
-                  label = messages("declaration.officeOfExit"),
-                  labelClass = Some("form-label-bold"),
-                  hintText = Some(messages("declaration.officeOfExit.hint")),
-                  messages("standard.officeOfExit.empty"),
-                  AutoCompleteItem.fromOfficeOfExit(OfficeOfExits.all)
-              )
-        @components.details(messages("site.details.summary_text_these")){
-            @tariffBody
-        }
-        @components.submit_button()
-        @components.buttons.save_and_return_later_button()
+        @govukFieldset(Fieldset(
+            legend = Some(Legend(
+                content = Text(messages("declaration.officeOfExit.title")),
+                classes = "govuk-fieldset__legend--xl",
```

Comment:
Can you use the constant Styles.gdsPageLegend here for the classes.

Reply:
Fixed!